services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -973,16 +973,23 @@
                   }
                 ]
               },
-              "dashboard": {
-                "title": "Add measures for monitoring access standards",
+              "verbiage": {
+                "dashboardTitle": "Add measures for monitoring access standards",
                 "addEntityButtonText": "Add access measure",
                 "editEntityButtonText": "Edit measure",
-                "deleteEntityButtonAltText": "Delete measure"
+                "addEditModalAddTitle": "Add access measure",
+                "addEditModalEditTitle": "Edit access measure",
+                "addEditModalMessage": "Complete the remaining indicators for this access measure by saving the measure and selecting \"Enter measure details\".",
+                "deleteEntityButtonAltText": "Delete measure",
+                "deleteModalTitle": "Delete access measure?",
+                "deleteModalConfirmButtonText": "Yes, Delete Measure",
```

Comment:
nit: are these all supposed to be capitalized?

Reply:
Ooh good catch. The capitalization on line 985 diverges from the designs. Changed.